x-pack/test/fleet_functional/apps/home/welcome.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const browser = getService('browser');
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'home']);
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Welcome interstitial', () => {
+    before(async () => {
+      // Need to navigate to page first to clear storage before test can be run
+      await PageObjects.common.navigateToUrl('home', undefined);
+      await browser.clearLocalStorage();
+      await esArchiver.emptyKibanaIndex();
+    });
+
+    /**
+     * When we run this against a Cloud cluster, we also test the case where Fleet server is running
+     * and ingesting elastic_agent data.
+     */
+    it('is displayed on a fresh install with Fleet setup executed', async () => {
```

Comment:
@liza-mae This is the test I'd like to execute on Cloud. Is there an easy way for me to try running this myself before I merge this?

Reply:
Ah nevermind, I remember that we have to get the changes merged in and built for a BC before we can test.